packages/webapp/src/webview/ui/components/App/App.scss

```diff
@@ -76,7 +76,7 @@ ul.striped-list > li:nth-of-type(odd) {
     padding-left: 0;
 }
 
-li {
+li.tree-item {
```

Comment:
@Klaus-Keller Maybe moving these css styles to the .tree-item class would be better. I have a branch with some other changes and I already included this fix https://github.com/SAP/guided-answers-extension/compare/feat/ga/91-content-tree-redesign#diff-e9eb3d4993dc04303a521ec8cd2307b6a564c07c68a99b3610163ddf7687d863R67

Reply:
No problem with me merging it now. It is not a big change anyway. Thanks